external/jacoco/README.md

```diff
@@ -0,0 +1,22 @@
+##External Jacoco Tests
```

Comment:
Please update to # External Jacoco Tests ( note there is a space between # and External. It's a format of Markdown).